doc/api/process.md

```diff
@@ -144,6 +144,59 @@ possible to record such errors in an error log, either periodically (which is
 likely best for long-running application) or upon process exit (which is likely
 most convenient for scripts).
 
+### Event: 'resolveSwallowed'
```

Comment:
non-blocking nit: not a fan of the name here. Not sure if I have a better suggestion tho. Perhaps just splitting these into two separate events? `process.on('resolveAfterResolved')` and `process.on('rejectAfterResolve')` or something similar?

Reply:
These mistakes are all very similar and that is the reason I implemented it as a single event. I also just [requested to add one more case](https://bugs.chromium.org/p/v8/issues/detail?id=8045) that could also just land in this event.

I am absolutely open to a new name tough!

Reply:
What about `ambiguousResolve`, `resolveOmission` or `variousResolves`?

Reply:
`multipleResolves` (or `redundantResolves`) would be better than `variousResolves`, imo.

Reply:
`redundantResolves` is a good choice, I think

Reply:
I personally like `multipleResolves`. Redundant sounds like that case could just be removed but it is likely that there is an actual bug in the code.

Reply:
resolveAfterResolve and rejectAfterResolve are pretty much as granular as you can get